services/indexes/avm/parser.go

```diff
@@ -5,16 +5,16 @@ package avm
 
 import (
 	"github.com/ava-labs/avalanchego/codec"
-	"github.com/ava-labs/avalanchego/vms/avm"
+	"github.com/ava-labs/avalanchego/vms/avm/txs"
 )
 
-func parseTx(c codec.Manager, bytes []byte) (*avm.Tx, error) {
-	tx := &avm.Tx{}
+func parseTx(c codec.Manager, bytes []byte) (*txs.Tx, error) {
```

Comment:
This function shouldn't exist anymore

Reply:
What should be used in its place?  Function is still called in a few places

Reply:
Initializing a new tx [Parser](https://github.com/ava-labs/avalanchego/blob/master/vms/avm/txs/parser.go#L40) and using [Parse( )](https://github.com/ava-labs/avalanchego/blob/master/vms/avm/txs/parser.go#L104) maybe? Does the same thing basically.

Reply:
Or this one, as recommended by Stephen: https://github.com/ava-labs/avalanchego/blob/master/wallet/chain/x/constants.go#L21